torch/utils/data/dataloader.py

```diff
@@ -30,6 +37,37 @@ def __init__(self, exc_info):
 _use_shared_memory = False
 r"""Whether to use shared memory in default_collate"""
 
+MANAGER_STATUS_CHECK_INTERVAL = 5.0
+
+if IS_WINDOWS:
+    class ManagerWatchdog(object):
```

Comment:
Here would be a good place for a comment discussing why the alternate ManagerWatchdog implementation is needed on Windows.